plover/formatting.py

```diff
@@ -776,6 +776,19 @@ def _apply_meta_attach(meta, ctx):
         new_word = add_suffix(last_word, meta)
         common_len = len(commonprefix([last_word, new_word]))
         replaced = last_word[common_len:]
+
+        if replaced.lower() != ctx.last_text(len(replaced)).lower():
```

Comment:
I think the code should check if the previous word was stitched, rather than assume the previous word was stitched if `replaced.lower() != ctx.last_text(len(replaced)).lower()`.

Reply:
Is it possible to check if the previous word was stitched if the stitched action is several behind?

Reply:
I can think of a lot of corner cases where getting the correct output would be problematic. 

Honestly, I don't think this feature is worth it: there's no practical use case, except for something like `S-P-E-L-L-I-N-G`, which can already be implemented with a  custom meta, so without the need for further complexity to the formatting code, and no expectations that it would work if followed by a suffix (which again make sense, I think).

Reply:
So do you think the plugin is sufficient and this should be closed?

Reply:
Yes.